dockerfiles/docker-compose.yml

```diff
@@ -38,14 +35,25 @@ services:
     command: ["./wait-for-it.sh", "db:5432", "--strict", "--", "./startup_script.sh"]
     restart: unless-stopped
 
+  extract-diffs:
+    image: makerdao/vdb-extract-diffs:latest # (needs wait-for-it.sh)
+    environment:
+      # Remote client would need to be statediff enabled
+      - CLIENT_IPCPATH=ws://geth-statediffing:8546
+      - STORAGEDIFFS_SOURCE=geth
+      - DATABASE_NAME=vdb
+      - DATABASE_HOSTNAME=db
+      - DATABASE_PORT=5432
+      - DATABASE_USER=vdb
+      - DATABASE_PASSWORD=vdb
+    depends_on:
+      - db
+    command: ["./wait-for-it.sh", "db:5432", "--strict", "--", "./startup_script.sh"]
+
   execute:
     image: makerdao/vdb-execute:latest # (needs wait-for-it.sh)
-    env_file: mainnet-1.0.1.env
```

Comment:
I think I'm missing something - how are we getting the contract addresses, etc in without this env file now?

Reply:
Moved to [config](https://github.com/makerdao/vdb-mcd-transformers/blob/staging/environments/docker.toml#L479) - apparently the actual deployment setup requires some cumbersome detail for env vars, so moving toward minimizing the details that need to remain in the env